cl/citations/tests.py

```diff
@@ -1727,3 +1735,222 @@ def test_get_graph_component(self):
                     sorted(output),
                     f"Got incorrect result from get_graph_component for inputs (expected {output}): {inputs}",
                 )
+
+
+class CitationLookUpApiTest(
```

Comment:
I think I'm calling for some pretty major changes, so I haven't reviewed the tests yet, but if a review would help at this stage, let me know and I'm happy to find a minute.